app/views/quotes/_quote.html.erb

```diff
@@ -31,6 +31,9 @@
       <div class="flex flex-col sm:flex-row text-gray-500">
         <div class="flex-grow"><%= quote.name %></div>
         <p><%= time_ago_in_words(quote.created_at) %> geleden.</p>
+        <%= if quote.reporter %>
```

Comment:
```
  <% if quote.reporter %>
```
De = geeft aan dat het vertaald moet worden naar HTML, hier moet alleen logic gebeuren dus dan hebben we dat niet nodig.